src/components/home/Home.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import LandingPage from './LandingPage';
+
+const Home = () => (
+  <main className="main--container w-100 h--100"
+    <section className="home--container h--100">
+      <LandingPage />
+    </section>
+  </main>
```

Comment:
### Hi @AbayomiOlaoye 

:diamonds: Please close this main tag in the src/components/home/Home.js page.